### PR TITLE
Ek

### PR DESCRIPTION
**Pull request by ekuiter**
added some publications.

also added some bibliographic data on our AMAI publication.

I am unsure to leave month and year as is? according to this page, the issue is formally from October 2024:
https://link.springer.com/journal/10472/volumes-and-issues/92-5

### CODE REVIEW

File: literature.bib [hunk 1]
```diff
@@ -4468,6 +4471,143 @@ @inproceedings{STS:VaMoS20
...
+	publisher = ACM,
+	address   = NY,
+	pages     = {98--102},
+	doi       = {10.1145/3336294.3342361},
+	year      = 2019
+}
+
+@inproceedings{GW:ICSE19,
+	author    = {Gazzillo, Paul and Wei, Shiyi},
+	title     = {Conditional Compilation is Dead, Long Live Conditional Compilation!},
```
Comment (SundermannC) [inline literature.bib:4563]: Double brackets missing for the title.

File: literature.bib [hunk 2]
```diff
@@ -4468,6 +4471,143 @@ @inproceedings{STS:VaMoS20
...
+	publisher = ACM,
+	address   = NY,
+	location  = Paris,
+	pages     = {130–-133},
+	isbn      = {9781450366687},
+	doi       = {10.1145/3307630.3342415},
+	year      = 2019
+}
+
+@inproceedings{DLKB:SPLC19,
```
Comment (SundermannC) [inline literature.bib:4550]: Put the SPLC19 entries in one batch?

File: literature.bib [hunk 3]
```diff
@@ -4468,6 +4471,143 @@ @inproceedings{STS:VaMoS20
...
+	year    = 2019
+}
+
+@inproceedings{NKSB:ESECFSE19,
+	author    = {Ne{\v{s}}i{\'{c}}, Damir and Kr{\"u}ger, Jacob and St{\u{a}}nciulescu, Stefan and Berger, Thorsten},
+	title     = {{Principles of Feature Modeling}},
+	booktitle = ESECFSE,
+	publisher = ACM,
+	address   = NY,
+	pages     = {62–-73},
```
Comment (SundermannC) [inline literature.bib:4532]: First - sign is wrong character.

File: literature.bib [hunk 4]
```diff
@@ -4468,6 +4471,143 @@ @inproceedings{STS:VaMoS20
...
+}
+
+@inproceedings{CCP+:SPLTea19,
+	author    = {Chavarriaga, Jaime and Casallas, Rubby and Parra, Carlos and Henao-Mej\'{\i}a, Martha Cecilia and Calle-Archila, Carlos Ricardo},
+	title     = {{Nine Years of Courses on Software Product Lines at Universidad de los Andes, Colombia}},
+	booktitle = SPLTea,
+	publisher = ACM,
+	address   = NY,
+	location  = Paris,
+	pages     = {130–-133},
```
Comment (SundermannC) [inline literature.bib:4544]: First - sign is wrong character.

**Review by SundermannC — CHANGES_REQUESTED** [review]
Some small problems that should be easy to fix.

File: literature.bib (hunk 1 above)
Comment (ekuiter) [reply literature.bib:4563]: thanks

File: literature.bib (hunk 4 above)
Comment (ekuiter) [reply literature.bib:4544]: I would have never noticed that

**Review by SundermannC — APPROVED** [review]

**Pull request merged**